provider/resources.go

```diff
@@ -348,9 +350,13 @@ func preConfigureCallbackWithLogger(ctx context.Context, host *provider.HostClie
 	return nil
 }
 
+//go:embed cmd/pulumi-resource-gcp/bridge-metadata.json
+var metadata []byte
+
 // Provider returns additional overlaid schema and metadata associated with the gcp package.
 func Provider() tfbridge.ProviderInfo {
 	p := shimv2.NewProvider(google.Provider())
+	p = pf.MuxShimWithPF(context.Background(), p, google.New(""))
```

Comment:
What's the significance of the `google.New("")` - why the empty string?

Reply:
It's a version. Perhaps version.Version would be more appropriate here, something we could try. 

Looks like the version is used by the pluginFrameworkProvider in particular in responses to GetMetadata function but I doubt we depend on that being accurate.

```go
// New is a helper function to simplify provider server and testing implementation.
func New(version string) provider.ProviderWithMetaSchema {
	return &frameworkProvider{
		version: version,
	}
}
```